### PR TITLE
Refine hide behavior

### PR DESCRIPTION
**Pull request by attila-at-hyper**

### CODE REVIEW

File: Source/CardStackController.m [hunk 1]
```diff
@@ -131,20 +131,23 @@ - (void)setIsSeachViewControllerHidden:(BOOL)isSeachViewControllerHidden
...
-        [self updateCardLocations];
+    if (isSeachViewControllerHidden) {
+        self.isOpen = NO;
+    }
+
+    POPSpringAnimation *springAnimation = [POPSpringAnimation animationWithPropertyNamed:kPOPViewFrame];
+    CGRect frame = self.searchViewController.view.frame;
+    frame.origin.y = isSeachViewControllerHidden ? -self.searchViewController.view.frame.size.height : 0;
+    springAnimation.toValue = [NSValue valueWithCGRect:frame];
+    springAnimation.springBounciness = 8;
```
Comment (3lvis) [inline Source/CardStackController.m]: Mind making a constant for the bounciness?

Comment (attila-at-hyper) [inline Source/CardStackController.m]: Done

**Comment by 3lvis** [comment]
Sweet!

**Comment by attila-at-hyper** [comment]
@3lvis: you rock! :)

**Pull request merged**